### PR TITLE
feat(Scripts): Adds a build:link script for use with npm link

### PR DESCRIPTION
**Pull request by larister**
Just a suggestion, it's fine if a) there's a better way or b) it's not deemed useful enough. It just took me a little while & a little help to figure out.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "build:css": "NODE_ENV=production BABEL_ENV=production webpack --config webpack.css.config.js",
     "build:static": "NODE_ENV=production BABEL_ENV=production webpack --config webpack.static.config.js",
+    "build:link": ". ./deploy/travis_setup.sh && npm run prepublish",
```
Comment (benji6) [inline package.json]: This should probably be `yarn prepublish` instead of `npm run prepublish`

Comment (HHogg) [inline package.json]: I agree with @benji6 comments about renaming from `link` and to use `yarn` 😉 ... also I wonder if `travish_setup.sh` needs renaming to something else like `prebuild.sh` ¯\_(ツ)_/¯ ...

**Comment by larister** [comment]
Brill thanks both, have updated accordingly.

**Comment by HHogg** [comment]
Thanks @larister. All is good to go except for the commit message, a `feat()` will trigger a new release so as it's purely a development enhancement it should really be a `chore()`

**Comment by larister** [comment]
Ah thanks @HHogg, fixed

**Pull request merged**